cruise.umple/test/cruise/umple/implementation/requirements/SortOneReq.ump

```diff
@@ -0,0 +1,10 @@
+// To test if a single requirement is outputted.
+req A01 {    
+  z requirement.
+}
+ implementsReq A01;   
+class Example { 
```

Comment:
The output of the requirement should indicate that A01 is implemented by class Example